src/hotspot/share/gc/g1/g1FullCollector.cpp

```diff
@@ -225,15 +228,20 @@ void G1FullCollector::complete_collection() {
   _heap->print_heap_after_full_collection(scope()->heap_transition());
 }
 
-void G1FullCollector::update_attribute_table(HeapRegion* hr) {
-  if (hr->is_free()) {
+void G1FullCollector::update_attribute_table(HeapRegion* hr, bool force_pinned) {
+  if (force_pinned) {
+    // Pin high live ratio region
```

Comment:
I think the comment is too specific as the argument name is `force_pinned` and not `is_high_live_ratio_region`. Please remove.